daemon/daemon.go

```diff
@@ -445,8 +445,8 @@ func (daemon *Daemon) setupResolvconfWatcher() error {
 			select {
 			case event := <-watcher.Events:
 				if event.Name == "/etc/resolv.conf" &&
-					(event.Op&fsnotify.Write == fsnotify.Write ||
-						event.Op&fsnotify.Create == fsnotify.Create) {
+					(event.Op&fsnotify.Write != 0 ||
+						event.Op&fsnotify.Create != 0) {
```

Comment:
these two can be put into one:
`event.Op & fsnotify.Write & fsnotify.Create != 0`

@LK4D4 please confirm